etl/app/server/profiles/catchmentCorrespondence.js

```diff
@@ -25,6 +26,43 @@ export const createQuery = `CREATE TABLE IF NOT EXISTS ${tableName}
     catchmentnhdplusid BIGINT
   )`;
 
+export const createIndexes = `
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitid_asc
+    ON ${tableName} USING btree
+    (assessmentunitid COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitname_asc
+    ON ${tableName} USING btree
+    (assessmentunitname COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
```

Comment:
The `catchmentnhdplusid` index is missing. In the local database, these are all null, so I'm not sure how it reflects in the full dataset, or if we already discussed this during a meeting.
```suggestion
CREATE INDEX IF NOT EXISTS ${indexTableName}_catchmentnhdplusid_asc
    ON ${tableName} USING btree
    (catchmentnhdplusid COLLATE pg_catalog."default" ASC NULLS LAST)
    TABLESPACE pg_default;
```